test/libsolidity/syntaxTests/array/call_uninitialized_function_through_array.sol

```diff
@@ -0,0 +1,16 @@
+// this used to be a semantics tests that was expected to fail, but is now caught at compile-time
```

Comment:
Not sure whether we really need to keep this one and the other one - but it's probably better to have too many tests than too few.